resources/styles/components/task-step/interactive.less

```diff
@@ -0,0 +1,13 @@
+@import '../../book-content/index';
+
+.task-step {
+
+  .interactive-step {
+    // pull in common stylings for book contents
+    .tutor-book-content();
+    iframe {
```

Comment:
Rather than mixin this twice without any changes, wouldn't it be better to just do a:
```css
.reading-step, .interactive-step {
.tutor-book-content();
}

```